pkg/crc/constants/constants_windows.go

```diff
@@ -9,8 +9,8 @@ const (
 	OcExecutableName            = "oc.exe"
 	PodmanExecutableName        = "podman.exe"
 	AdminHelperExecutableName   = "admin-helper-windows.exe"
-	TrayExecutableName          = "tray-windows.exe"
-	TrayShortcutName            = "tray-windows.lnk"
+	TrayExecutableName          = "crc-tray.exe"
+	TrayShortcutName            = "crc-tray.lnk"
```

Comment:
Has there been a release with these changes? Seems like this would break `crc.exe setup` with the tray enabled until there is a new release?

Reply:
hence I would not merge... but this change is needed...
... really depends on if we bump the release. I am not inclined at the moment.

Reply:
@anjannath any thoughts on this?

Reply:
@gbraad Close this in favor of #2290 ? that handles the upgrade scenario!

Reply:
New release has been published with the changed name: https://github.com/code-ready/tray-windows/releases/tag/v0.5.0.0